packages/site/src/layout/NavMenu.module.css

```diff
@@ -1,34 +1,34 @@
 .navMenuContainer {
-    --hoverOffset: var(--space-smallest);
-    --negativeHoverOffset: calc(-1 * var(--hoverOffset));
-    --subItemInset: calc(var(--space-small) + var(--space-smaller));
-    --navItemHeight: 40px;
-    display: flex;
-    flex-direction: column;
-    padding: 36px 0 0 0;
-    height: 100dvh;
-    width: 200px;
-    box-sizing: border-box;
+  --hoverOffset: var(--space-smallest);
+  --negativeHoverOffset: calc(-1 * var(--hoverOffset));
+  --subItemInset: calc(var(--space-small) + var(--space-smaller));
+  --navItemHeight: 40px;
+  display: flex;
+  flex-direction: column;
+  padding: 36px 0 0 0;
+  height: 100dvh;
+  width: 200px;
+  box-sizing: border-box;
 }
 
-.navMenuContainer > * {
+.navMenuContainer>* {
```

Comment:
The other formatting changes in this file look good but I think this one needs to be reverted

Reply:
Huh, I wonder why your editor is changing that, it is definitely less readable. You don't need any spaces in CSS so  `.navMenuContainer>*{padding-left: var(--space-base);padding-right: var(--space-base);}` would still work but I think the consensus out there is that `.navMenuContainer > *` is preferable. Anyways, I will approve it anyways as this is the only thing I came across but worth looking into your editor at some point

Reply:
@Aiden-Brine I figured it out! Just pushed a new commit & will merge after. This has been bugging me but thanks for pushing me to find a solution 😅 

<img width="688" alt="Screenshot 2024-12-16 at 5 52 50 PM" src="https://github.com/user-attachments/assets/15ddead5-1fb3-4529-9809-68d65aece9b0" />

Reply:
Nice! Thanks for doing that